lib/excalt/vcard/helpers.ex

```diff
@@ -0,0 +1,3 @@
+defmodule Excalt.Vcard.Helpers do
+
```

Comment:
What do you want to put here? If nothing for the moment I'd suggest to remove the file.

Reply:
Populated the file with auth function.